lib/clean.js

```diff
@@ -46,6 +46,17 @@ var CleanCSS = module.exports = function CleanCSS(options) {
   };
 };
 
+
+// for compatibility with optimize-css-assets-webpack-plugin
+CleanCSS.process = function (input, opts) {
+  var cleanCss = new CleanCSS(Object.assign({ returnPromise: true }, opts));
+  return cleanCss.minify(input)
+         .then(function(output) {
+           return { css: output.styles };
+         });
```

Comment:
Cool. Please use 2-space indent in lines 54-56 to be consistent with other code here and I'll merge it, thanks!